CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* Fix Inputs background in darkTheme
```

Comment:
There is no need to update the CHANGELOG as it will be automatically updated upon the release

Reply:
Fixed